Lib/tkinter/ttk.py

```diff
@@ -1643,7 +1643,10 @@ def set_menu(self, default=None, *values):
         menu.delete(0, 'end')
         for val in values:
             menu.add_radiobutton(label=val,
-                command=tkinter._setit(self._variable, val, self._callback),
+                command=(
+                    None if self._callback is None
+                    else lambda val=val: self._callback(val)
+                ),
```

Comment:
I am not sure that `command=None` works. Needs testing.

Reply:
Works well in the quick tests I've done.

Reply:
This is not a problem. [`__init__.py` line 1476](https://github.com/python/cpython/blob/778b07565e38/Lib/tkinter/__init__.py#L1504) filters kwargs with a value of `None`.